docs/routing.md

```diff
@@ -144,7 +144,36 @@ route.json
       "host": "hello-openshift.v3.rhcloud.com",
       "serviceName": "hello-openshift"
     }
-    
+
+## Securing Your Routes
+
+Creating a secure route to your pods can be accomplished by specifying the TLS Termination of the route and, optionally,
+providing certificates to use.  TLS Termination falls in the following configuration buckets:
+
+#### Edge Termination
+Edge termination means that TLS Termination occurs prior to traffic reaching the pod.  TLS certificates are served by either 
+inspecting the SNI host sent in the request and finding a certificate for that host (matching the `CommonName` in the 
```

Comment:
I don't follow, if we always use a generic certificate then we aren't really offering custom ssl edge termination with a user's certificate.  Can you describe in more detail what you're thinking?  This is the flow that I'm currently trying to achieve: https://docs.google.com/a/redhat.com/drawings/d/1fRsvKg1iuY1EfQ-C3O6s8lxxouapzuCwVz7jFUKDUsc/edit

Full disclosure: I'm getting intermittent failures from haproxy today in my edge term testing that I have yet to explain so I'm not opposed to a pivot here.